src/compile.coffee

```diff
@@ -222,4 +218,26 @@ compilePathOrigin = (filename, parseResult, httpTransaction) ->
   }
 
 
+compileOriginExampleName = (parseResult, mediaType, httpTransaction) ->
+  exampleName = ''
+
+  transition = parent(httpTransaction, parseResult, {element: 'transition'})
+  httpResponse = child(httpTransaction, {element: 'httpResponse'})
+
+  if mediaType is 'text/vnd.apiblueprint'
```

Comment:
I'd move `exampleName = ''` just above the if. No big deal, but it would help me in orientation.